src/api/lookahead.rs

```diff
@@ -44,6 +45,9 @@ pub(crate) fn estimate_intra_costs<T: Pixel>(
   let w_in_imp_b = plane.cfg.width / IMPORTANCE_BLOCK_SIZE;
   let mut intra_costs = Vec::with_capacity(h_in_imp_b * w_in_imp_b);
 
+  let mut edge_buf: Aligned<[T; 4 * MAX_TX_SIZE + 1]> =
+    Aligned::from_fn(|_| T::zero());
```

Comment:
There is a terse pattern for fully initialized `Aligned<[T; N]>`:
```suggestion
  let mut edge_buf = Aligned::new([T::zero(); 4 * MAX_TX_SIZE + 1]);
```